config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   resources :votes
+  resources :choices do
+    patch "count_up", on: :member
+  end
```

Comment:
[want]
`resources`+`member`だと不要なCRUDが発生するので
以下のように書き換えることで対応できそう。
```suggestion
patch 'choices/:id', to: 'choices#count_up', as: 'count_up_choice'
```